templates/base.html

```diff
@@ -44,28 +44,27 @@
                         <!-- Nav -->
                         <div class="show-for-small-only">
                             <ul class="nav">
-                                <li><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
+                                <li class="small-menu"><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
                             </ul>
                         </div>
                         <div class="hide-for-small-only">
                             <ul class="nav">
                                 <li><a class="active" href="{% url 'home' %}">HOME</a></li>
                                 <li><a class="active" href="{% url 'artists' %}">ARTISTS</a></li>
                                 {% if user.is_authenticated %}
-                                    <li style="float: right;"><a class="active" href="{% url 'logout' %}">LOGOUT</a></li>
-                                    <li style="float: right;"><a class="active" href="{% url 'profile' %}">PROFILE</a></li>
+                                    <li style="float: right;"><a href="{% url 'logout' %}">LOGOUT</a></li>
```

Comment:
I like that you got rid of the extra `"active"` classes here. Should we get rid of the one for the `ARTISTS` link too?